etc/skel/.config/hypr/config/keybinds.conf

```diff
@@ -28,8 +28,8 @@ bindd = $mainMod, G, Remove gaps between window, exec, hyprctl --batch "keyword
 
 # ======= Volume Control =======
 
-bindde = , XF86AudioLowerVolume, Decreases player audio volume, exec, pamixer -ud 3 && pamixer --get-volume > /tmp/$HYPRLAND_INSTANCE_SIGNATURE.wob
-bindde = , XF86AudioRaiseVolume, Increases player audio volume, exec, pamixer -ui 3 && pamixer --get-volume > /tmp/$HYPRLAND_INSTANCE_SIGNATURE.wob
+bindd = , XF86AudioLowerVolume, Decreases player audio volume, exec, pamixer -ud 3 && pamixer --get-volume > /tmp/$HYPRLAND_INSTANCE_SIGNATURE.wob
+bindd = , XF86AudioRaiseVolume, Increases player audio volume, exec, pamixer -ui 3 && pamixer --get-volume > /tmp/$HYPRLAND_INSTANCE_SIGNATURE.wob
```

Comment:
These ones too xD

Reply:
Damnit xD!
Also did it for `XF86AudioMute` just in case lol.

Reply:
Mute doesn't need `e`. It's not supposed to repeat when hold (intuitively).

Reply:
Unless you want to use it as a music effect, think about it.